### PR TITLE
Add Helm charts tutorial article

### PR DESCRIPTION
**Pull request by bittner**
Adds a tutorial on deploying Helm charts on APPUiO.

We get quite a few support requests about this topic, hence it makes sense to describe a common approach here to be able to simply point to our documentation in future.

**Internal reference:** APPU-2544

### CODE REVIEW

File: app/helmcharts.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    configured with database-optimized settings.
+
+Compatibility
+-------------
+
+Keep in mind that some Helm charts might not be compatible with OpenShift.
+This is mostly due to the more strict security measures OpenShift enforces.
+The easiest way to determine OpenShift compatibility is to install a certain
+chart and see if everything works. If not, you'll have to investigate pods
+that won't start deeper or find another chart.
```
Comment (mhutter) [inline app/helmcharts.rst]: Maybe mention https://docs.openshift.com/container-platform/3.11/creating_images/guidelines.html#openshift-specific-guidelines

**Review by mhutter — CHANGES_REQUESTED** [review]

**Review by srueg — APPROVED** [review]

**Comment by tobru** [comment]
LGTM

File: app/helmcharts.rst (hunk 1 above)
Comment (bittner) [reply app/helmcharts.rst]: Good point! I added a paragraph with that link.

**Comment by bittner** [comment]
I incorporated the changes suggested by @mhutter and rebased to PR (pulling in the changes from #66). Second round of review, please! :eyes:

**Review by srueg — APPROVED** [review]

**Pull request merged**